tox.ini

```diff
@@ -0,0 +1,7 @@
+[tox]
+envlist=py27,py35,py36,py37
+
+[testenv]
+changedir=tests
```

Comment:
should this be pytests?

Reply:
I looked in `CONTROBUTING.md` for the directory of pytests so I found tests was the one.
`py -3.7 -m tests -v`.